### PR TITLE
Docs for Habitica component

### PR DESCRIPTION
**Pull request by asmfreak**
**Description:**
Documentation for Habitica component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#15744

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. (Forked from `next`)
- [x] The documentation follow the [standards][standards].
- [X] Documentation for component. 
- [x] Documentation for sensor platform.
- [x] Fix changes in configuration from code review

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/habitica.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+title: "Habitica"
+description: "Instructions on enabling Habitica support for your Home Assistant"
+date: 2018-08-09 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: habitica.png
+ha_category: Hub
+ha_version: 0.76
```
Comment (MartinHjelmare) [inline source/_components/habitica.markdown]: 0.78

File: source/_components/habitica.markdown [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+---
```
Comment (MartinHjelmare) [inline source/_components/habitica.markdown]: Please add `ha_iot_class`.

File: source/_components/habitica.markdown [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+```yaml
+# Minimum viable configuration.yaml entry
+habitica:
+   - api_user: 'REPLACE_ME_WITH_YOUR_USER_ID'
+     api_key: 'REPLACE_ME_WITH_YOUR_API_KEY'
+```
+
+You can specify several users, providing `api_user` and `api_key` for each.
+At runtime you will be able to use api for each respective user by their Habitica's username.
+You can override this by passing `name` key, this value will be used istead of the username.
```
Comment (MartinHjelmare) [inline source/_components/habitica.markdown]: will be used instead of

File: source/_components/sensor.habitica.markdown [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+description: "Instructions on enabling Habitica support for your Home Assistant"
+date: 2018-08-09 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: habitica.png
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_version: 0.76
```
Comment (MartinHjelmare) [inline source/_components/sensor.habitica.markdown]: 0.78

File: source/_components/habitica.markdown [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+This component allows you to monitor and manage your Habitica profile. This component exposes the [Habitica's API](https://habitica.com/apidoc/) as a Home Assistant service. It support multple users and allows you to automate checking out your habits and daily tasks or casting magics using Home Assistant.
+This component enables usage of [`sensor.habitica` platform.](/components/sensor.habitica/)
+To use the component you should use this example configuration:
+
+```yaml
+# Minimum viable configuration.yaml entry
+habitica:
+   - api_user: 'REPLACE_ME_WITH_YOUR_USER_ID'
```
Comment (MartinHjelmare) [inline source/_components/habitica.markdown]: Remove a space before the dash. We use 2 space indentation in the yaml.

File: source/_components/habitica.markdown [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+comments: false
+sharing: true
+footer: true
+logo: habitica.png
+ha_category: Hub
+ha_version: 0.78
+ha_iot_class: "Cloud Polling"
+---
+
+This component allows you to monitor and manage your Habitica profile. This component exposes the [Habitica's API](https://habitica.com/apidoc/) as a Home Assistant service. It support multiple users and allows you to automate checking out your habits and daily tasks or casting magics using Home Assistant.
```
Comment (MartinHjelmare) [inline source/_components/habitica.markdown]: It supports multiple users...

**Comment by asmfreak** [comment]
bump?

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**